src/app/api/(checkout)/create-intent/[orderId]/route.ts

```diff
@@ -15,7 +16,7 @@ export const POST = async({ params }: {params: { orderId: string}}) => {
     if(order){
         const paymentIntent = await stripe.paymentIntents.create({
             amount: 100 * 100,
-            currency: "gbp",
+            currency: "usd",
```

Comment:
Let's create an enum for all currency values

Reply:
okay will do